measures/2020/measures-schema.yaml

```diff
@@ -1,11 +1,21 @@
-$id: https://github.com/CMSgov/qpp-measures-data/blob/master/measures/2019/measures-schema.yaml
+$id: https://github.com/CMSgov/qpp-measures-data/blob/master/measures/2020/measures-schema.yaml
 $schema: http://json-schema.org/draft-07/schema
 type: array
 items: { $ref: #/definitions/measure }
 uniqueItemProperties: ['measureId'] # used by ajv-keywords
 
 definitions:
   measure:
+    title: 'Measure'
+    type: object
+    anyOf:
```

Comment:
I wonder if this should be an `anyOf` or a `oneOf`?

Reply:
In practice it shouldn't be possible for an object to match multiple of these schemas since each one has `additionalProperties: false` and each one also has a different const value of `category`. `anyOf` will be faster since it will stop validating once the data matches any of them. Since this schema is only used by unit tests, though, speed isn't really a concern at all. No strong feeling either way for me

Reply:
Sounds good then